src/plugins/ncr/reducers/nodes.js

```diff
@@ -27,6 +28,16 @@ const onPbjxEnvelopeReceived = (prevState, action) => {
   return state;
 };
 
+const onNodesRecieved = (prevState, action) => {
+  const state = { ...prevState };
+  action.nodes.forEach((node) => {
+    state[node.schema().getCurie().getMessage()] = { ...state[node.schema().getCurie().getMessage()] };
```

Comment:
@nanetteCodes bit easier to read if you just assign `node.schema().getCurie().getMessage()` to `message` at beginning of loop

Reply:
updated to use message variable